README.md

```diff
@@ -51,12 +51,8 @@ some kind of discriminant:
 
 ```ts
 // enum declarations
-enum Colors {
-  red,
-  green,
-  blue
-}
 
+// Implicitly extends `Number`, auto-increments values by 1 starting at 0
```

Comment:
why would this implicitly extend Number, or is that the default?

Reply:
`Number` is the default, per the Remarks section.